### PR TITLE
Add window webworker checks to Common (Basic SKU)

### PR DESCRIPTION
**Pull request by markwolff**

### CODE REVIEW

File: AISKULight/rollup.config.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ const browserRollupConfigFactory = (isProduction, libV = '1') => {
       file: `browser/aib.${libV}.js`,
       banner: banner,
       format: "umd",
-      name: "aibasic",
+      name: "Microsoft.Azure",
```
Comment (jpiyali) [inline AISKULight/rollup.config.js]: does this create all types in namespace Microsoft.Azure?

Comment (markwolff) [reply AISKULight/rollup.config.js]: Only when not using a loader, and just referencing global namespace. For AISKU we manually modify window so it is at both places. It is a required field in rollup for bundling as umd.

Comment (jpiyali) [reply AISKULight/rollup.config.js]: but this is the rollup for browser (umd?)

Comment (markwolff) [reply AISKULight/rollup.config.js]: yes, it is the fallback for if no cjs or amd
```js
(function (global, factory) {
    typeof exports === 'object' && typeof module !== 'undefined' ? factory(exports) :
    typeof define === 'function' && define.amd ? define(['exports'], factory) :
    (factory((global.Microsoft = global.Microsoft || {}, global.Microsoft.Azure = {})));
}(this, (function (exports) { 'use strict';
```

**Pull request merged**